cmd/gossip.go

```diff
@@ -0,0 +1,192 @@
+package main
```

Comment:
I suppose I could just keep this locally. In any case, I prefer having this test case inside this repo when investigating issues because it is easier to add debugging logs and make code changes to `aw` instead of needing to use go mod in `awt` after each change.